### PR TITLE
Explanation about the meaning of date parameter

### PR DESCRIPTION
**Pull request by gtoonstra**
Proposal for updating the tutorial doc, related to issue #87

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -373,6 +375,10 @@ Note that if you use ``depends_on_past=True``, individual task instances
 will depend on the success of the preceding task instance, except for the
 start_date specified itself, for which this dependency is disregarded.
 
+The date range in this context is a ``start-date`` and optionally an ``end-date``,
```
Comment (mistercrunch) [inline docs/tutorial.rst]: use `_` for `start_date` and `end_date` to match `airflow backfill -h` 's params

File: docs/tutorial.rst [hunk 2]
```diff
@@ -373,6 +375,10 @@ Note that if you use ``depends_on_past=True``, individual task instances
...
+which are both passed to the scheduler and used to populate the run schedule
```
Comment (mistercrunch) [inline docs/tutorial.rst]: "which are used to execute a backfill on this date range. The backfill command proceed by submitting task instances to the executor while respecting the dependencies as they evolve."

This is a technicality, but the backfill process doesn't talk to the scheduler process at any point in time.

**Comment by gtoonstra** [comment]
Changes made.

**Comment by mistercrunch** [comment]
img

**Pull request merged**